### PR TITLE
chore: add schema validation errors when possible

### PR DESCRIPTION
**Pull request by menduz**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -97,8 +97,7 @@ export type Validation = {
  * @public
  */
 export type ConditionalValidation = {
-  predicate: (args: ValidationArgs) => boolean | Promise<boolean>
-  message: (args: ValidationArgs) => string
```
Comment (agusaldasoro) [inline src/types.ts:101]: why is `message` deleted?

**Review by agusaldasoro — APPROVED** [review]

**Comment by agusaldasoro** [comment]
Please remove package-lock.json file as we are using yarn

File: src/types.ts (hunk 1 above)
Comment (menduz) [reply src/types.ts:101]: because it needs to come from three lines above to not lose the error messages!

**Comment by menduz** [comment]
Sorry, it is done @agusaldasoro

**Pull request merged**